alexandra/app.py

```diff
@@ -1,13 +1,12 @@
 import logging
+import inspect
```

Comment:
Switched to inspect because it seems to be a more straightforward and sensible way to get function arg counts